shell/src/main/java/alluxio/cli/fsadmin/command/report/SummaryCommand.java

```diff
@@ -32,24 +36,16 @@
  * Print Alluxio cluster summarized information.
  * This class depends on meta master client and block master client.
  */
-public class SummaryCommand {
-  /** Print Alluxio cluster summarized information. */
-  public static void printSummary() {
+public class SummaryCommand extends AbstractCommand {
```

Comment:
Summary is a mode for the report command, not a command itself, so it shouldn't implement `Command`

Reply:
fixed, thanks